NRLMMD-GEOIPS/recenter_tc
# updated recenter_tc to use output_checkers interface

See https://github.com/NRLMMD-GEOIPS/geoips/pull/351 for more information and reviewer checklist. Still need to update release notes for this PR.

# Related Issues
fixes NRLMMD-GEOIPS/geoips#293

# Testing Instructions
Run GeoIPS test script geoips/tests/integration_tests/full_test.sh to ensure nothing has gone wrong. Should end up with a 0 as a result.

# Summary
This PR is an extension of the GeoIPS PR 'Convert compare_outputs.py into an 'output_checkers' interface'. We needed to modify recenter_tc to use that interface instead of the old compare_outputs.py module.
